app/src/main/java/com/github/libretube/util/NowPlayingNotification.kt

```diff
@@ -129,6 +137,27 @@ class NowPlayingNotification(
         mediaSession.isActive = true
 
         mediaSessionConnector = MediaSessionConnector(mediaSession)
+        mediaSessionConnector.setQueueNavigator(object : TimelineQueueNavigator(mediaSession) {
+            override fun getMediaDescription(
+                player: Player,
+                windowIndex: Int
+            ): MediaDescriptionCompat {
+                return MediaDescriptionCompat.Builder().apply {
+                    setTitle(streams?.title!!)
+                    setSubtitle(streams?.uploader)
+                    val extras = Bundle()
+                    val appIcon = BitmapFactory.decodeResource(
+                        Resources.getSystem(),
```

Comment:
`Resources.getSystem()` only accesses the system-specific resources, but it can't access LibreTube internal resources such as an app icon, so changing that line might fix the app icon :)